socorro/cron/jobs/clean_raw_adi.py

```diff
@@ -0,0 +1,34 @@
+from configman import Namespace
+from crontabber.base import BaseCronApp
+from crontabber.mixins import (
+    with_postgres_transactions,
+    with_single_postgres_transaction,
+)
+
+
+@with_postgres_transactions()
+@with_single_postgres_transaction()
```

Comment:
Nope and I'm not interested. I manually deleted all the old cruft. 
This new job is going to run every day so the amount of stuff it deletes is only going to be a days worth.